cmd/update.go

```diff
@@ -55,11 +57,10 @@ func isAborted(input string) (aborted bool, err error) {
 
 func selfUpdate() error {
 	latestVersion, err := canUpdate()
-	if err != nil {
+	if latestVersion.AssetURL == "" {
 		return err
 	}
```

Comment:
```suggestion
	if err != nil {
		return err
	}
	if latestVersion.AssetURL == "" {
		return fmt.Errorf("Asset URL is empty")
	}
```

Reply:
Thanks for the review.
The errors were mixed, so I separated.